doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -373,7 +374,19 @@ Nordic Security Module
 Other libraries
 ---------------
 
-|no_changes_yet_note|
+* :ref:`lib_identity_key` library:
+
+  * Updated:
+
+    * :c:func:`identity_key_write_random`, :c:func:`identity_key_write_key` and :c:func:`identity_key_write_dummy` functions to return an error code and not panic on error.
+    * :c:func:`identity_key_read` function to always return an error code from the library-defined codes.
+
+* :ref:`lib_hw_unique_key` library:
+
+  * Updated:
+
+    * :c:func:`hw_unique_key_write`, :c:func:`hw_unique_key_write_random` and :c:func:`hw_unique_key_load_kdr` functions to return an error code and not panic on error.
+    * :c:func:`hw_unique_key_derive_key` function to always return an error code from the library-defined codes and not forward error codes from underlying libraries.
```

Comment:
Nit: Strange to use different wording with the above, which is the same change. Maybe align them?

Reply:
Done